src/components/inputs/Select.vue

```diff
@@ -1,9 +1,11 @@
 <template>
-	<select v-model="vmodelProxy" class="select">
-		<option v-for="option in options" :key="option.value" :value="option.value">
-			{{ option.label }}
-		</option>
-	</select>
+	<div>
```

Comment:
We should use a `label` Tag for labels in all the inputs. (like in the [SC components](https://github.com/schul-cloud/nuxt-client/blob/develop/src/components/base/BaseInput/BaseInputRadio.vue))